code/modules/mob/living/simple_animal/abnormality/he/helper.dm

```diff
@@ -52,6 +52,8 @@
 	chosen_attack_num = 1
 
 /mob/living/simple_animal/hostile/abnormality/helper/AttackingTarget()
+	if(QliphothDeterrence())
```

Comment:
See, this is why I told you to add a check to click proc and not the other things.
AI controlled abnormalities already can't do anything before breach, so we only need to check for player input.